src/app/views/schedule/Schedule.jsx

```diff
@@ -6,22 +6,26 @@ import {
 } from "src/app/components";
 
 import "./Schedule.scss";
+import { Container } from "react-bootstrap";
 
-const HACKING_DEADLINE = "28 May 2023 09:00:00 PDT";
+const HACKING_DEADLINE = "26 May 2024 09:00:00 PDT";
 
 const Schedule = () => (
 	<div className="Schedule">
 		<section id="schedule-block">
 			<div className="schedule-header">
-				<FloatingLaptop offset={0} />
-				<h2>Schedule</h2>
-				<FloatingLaptop offset={2} />
+				{/* <FloatingLaptop offset={0} /> */}
```

Comment:
ehh let's just get rid of the commented floating things if we're not going to use them